### PR TITLE
Fix //jumpto

### LINKED ISSUE #548: jumpto + coordinate not working
# Bug report for FastAsyncWorldEdit 1.16.1

If I use `/jumpto` all works fine.
But with `/jumpto [world,x,y,z]` I get the error: "error while handling a WorldEdit command"

https://pastebin.com/KsEfh6uP

**[REQUIRED] FastAsyncWorldEdit Configuration Files:** 
https://athion.net/ISPaster/paste/view/9d90979f779e4bb3bda8285de5b0d17a

**[REQUIRED] FastAsyncWorldEdit Version Number:** 
> fawe 1.16 - 306

**[REQUIRED] Spigot/Paper Version Number:** 
> PaperMC 1.15.2

Currently, I have not a testserver for 1.16 ^^

**How to replicate**:
`/jumpto world;12;12;12`
`/jumpto world,12,12,12`

**Checklist**:

- [X] I included all information required in the sections above
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit/issues?q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of [FastAsyncWorldEdit for 1.16.1](https://ci.athion.net/job/FastAsyncWorldEdit-1.16/)
- [X] I made sure the bug/error is not caused by any other plugin

### PR DESCRIPTION
**Pull request by weaondara**
## Overview
added location argument parser

**Fixes #548**

## Description
added location argument parser

## Checklist

- [x] I included all information required in the sections above
- [x] I tested my changes and approved their functionality
- [x] I ensured my changes do not break other parts of the code
- [x] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/FastAsyncWorldEdit/blob/1.16/CONTRIBUTING.md)

### CODE REVIEW

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/argument/LocationConverter.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    @Override
+    public Component describeAcceptableArguments() {
+        return desc;
+    }
+
+    @Override
+    public List<String> getSuggestions(String input, InjectedValueAccess context) {
+        if (input.contains(","))
```
Comment (SirYwell) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/argument/LocationConverter.java:60]: Missing braces

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/argument/LocationConverter.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        String[] split4 = s.split(",", 4);
+        if (split4.length != 4) {
+            return FailedConversion.from(new IllegalArgumentException(
+                    "Must have exactly 1 world and 3 vector components"));
+        }
+
+        String[] split2 = s.split(",", 2);
+
+        ConversionResult<World> world = worldConverter.convert(split2[0], injectedValueAccess);
+        if (!world.isSuccessful())
```
Comment (SirYwell) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/argument/LocationConverter.java:76]: Missing braces

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/argument/LocationConverter.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            return (FailedConversion) world;
+        ConversionResult<BlockVector3> vector = vectorConverter.convert(split2[1], injectedValueAccess);
+        if (!vector.isSuccessful())
```
Comment (SirYwell) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/argument/LocationConverter.java:79]: Missing braces

**Review by SirYwell — CHANGES_REQUESTED** [review]
Besides the minor checkstyle stuff:
I think it might be worth to have the world as an optional parameter, don't know what you and the others think about that.

**Review by MattBDev — APPROVED** [review]
Address the issue with the if statement braces and it will be ready to be merged.

**Review by aurorasmiles — APPROVED** [review]

**Review by N0tMyFaultOG — APPROVED** [review]

**Pull request merged**